src/ellipticLCS.jl

```diff
@@ -652,21 +724,37 @@ function ellipticLCS(T::AbstractMatrix{SymmetricTensor{2,2,S,3}},
 end
 function ellipticLCS(T::AxisArray{SymmetricTensor{2,2,S,3},2},
                         p::LCSParameters=LCSParameters();
-                        outermost::Bool=true,
-                        verbose::Bool=true,
                         debug::Bool=false,
-                        singularity_predicate=nothing) where S <: Real
+                        unique_vortices=true,
+                        singularity_predicate=nothing,
+                        kwargs...) where S <: Real
     # detect centers of elliptic (in the index sense) regions
-    xspan = T.axes[1]
-    xmax = xspan[end]
-    singularities = singularity_detection(T, p.indexradius; combine_pairs=p.combine_pairs,combine_31=p.combine_31)
+    singularities = singularity_detection(T, p.indexradius; merge_heuristics=p.merge_heuristics)
     if singularity_predicate != nothing
```

Comment:
Checking for `nothing` is done via `=== nothing`, checking for not-nothing is via `!== nothing`.